code401challenges/src/main/java/code401challenges/hashtable/RepeatedWord.java

```diff
@@ -1,4 +1,35 @@
 package code401challenges.hashtable;
 
+import com.google.common.base.Strings;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 public class RepeatedWord {
-}
+
+    public static void main(String[] args) {
+        System.out.println();
+    }
+
+
+    public  HashTable repeatedWord(String str) {
+        HashTable ht = new HashTable(1024);
+        String[] splitWords = str.split(" ");
+        Set<Strings>  words = new HashSet(Arrays.asList(splitWords));
+        int count = 0;
+        for(Strings word :  words) {
```

Comment:
There is a way to solve this in using one loop instead of a nested loop. If you have time I encourage you to try to refactor this to be more efficient